protostar/cli/signable_command_util.py

```diff
@@ -1,97 +1,73 @@
 import importlib
 import os
-from typing import Any, Type
+from typing import Any, Optional, Type, Union
 
 from starknet_py.net.signer import BaseSigner
 from starknet_py.net.signer.stark_curve_signer import KeyPair, StarkCurveSigner
 
 from protostar.protostar_exception import ProtostarException
 from protostar.starknet_gateway import NetworkConfig
-from .protostar_argument import ProtostarArgument
+from protostar.starknet import Address
+from protostar.cli.common_arguments import PRIVATE_KEY_ENV_VAR_NAME
 
-PRIVATE_KEY_ENV_VAR_NAME = "PROTOSTAR_ACCOUNT_PRIVATE_KEY"
 
+def get_signer(
+    args: Any,
+    network_config: NetworkConfig,
+    account_address: Optional[Union[Address, int, str]],
```

Comment:
We can assume it is there if it is required by command, we also assume it for `signer_class` as well.
I suggest the type
```
account_address: Address
```
and using `from_user_input` constructor. It will be more consistent and `Address` type supports conversion to int.

Reply:
Changed to Optional[Address] to split account_address validation (in Address class) from get_signer logic.
Optional is required as account_address may not be necessary if args.signer_class is provided.